test/test_helper.bash

```diff
@@ -13,6 +13,15 @@ fixtures() {
   TEST_RELATIVE_FIXTURE_ROOT=$(bats_trim_filename "${TEST_FIXTURE_ROOT}" TEST_RELATIVE_FIXTURE_ROOT)
 }
 
+sudo_cmd() {
```

Comment:
IMHO a bats_sudo that wraps the call to sudo would be more readable.

Reply:
Makes sense. I changed the function to be a wrapper.